### PR TITLE
chore(tab-nav): adds base message bundles

### PR DESCRIPTION
**Pull request by Elijbet**
**Related Issue:** #6689

## Summary

This will add base message bundles to enable the translations build process.

### CODE REVIEW

**Review by anveshmekala — APPROVED** [review]
👍 

This is low-risk.

File: packages/calcite-components/src/components/tab-nav/assets/tab-nav/t9n/messages.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "nextTabTitles": "Next tab titles",
```
Comment (jcfranco) [inline packages/calcite-components/src/components/tab-nav/assets/tab-nav/t9n/messages.json:2]: +@geospatialem @SkyeSeitz @ashetland for editorial review. We can make edits in a follow-up PR for October.

**Review by jcfranco — APPROVED** [review]
💥

**Review by jcfranco — COMMENTED** [review]
Sorry for not catching this sooner, but you also need to update `t9nmanifest.txt` for these to get picked up for translations.

Good to merge afterwards.

**Review by jcfranco — APPROVED** [review]

**Comment by jcfranco** [comment]
Merging to expedite for translation since these shouldn't have an impact on E2E tests.

**Pull request merged**